lib/apache

```diff
@@ -206,6 +234,12 @@ function disable_apache_site {
         if [[ -f ${enabled_site_file} ]]; then
             sudo mv ${enabled_site_file} ${enabled_site_file}.disabled
         fi
+    elif is_clearlinux; then
```

Comment:
I don't see any differences here from the is_fedora case, just add || is_clearlinux to line 231?